.github/workflows/packaging-test.yml

```diff
@@ -1,7 +1,13 @@
 name: Packaging Tests
 
 on:
+  push:
+    branches:
+    - main
```

Comment:
Given https://github.com/scikit-hep/awkward/pull/3012#discussion_r1483499237 kept PRs and schedule, but added push events. Can be removed.